buildchain/buildchain/vagrant.py

```diff
@@ -40,16 +39,60 @@ def mkdir_dot_vagrant() -> None:
         'uptodate': [True],
     }
 
-def task_vagrantup() -> types.TaskDict:
+def task__vagrant_up_noprov() -> types.TaskDict:
+    """Run `vagrant up` without provisioning a development environment."""
+    cmd = [config.VAGRANT, 'up']
+    cmd.extend(map(shlex.quote, config.VAGRANT_UP_ARGS))
+    cmd.extend(['--no-provision'])
```

Comment:
minor: by using `append` instead of `extend` you don't need to wrap the string into a list.